### DISCUSSION TITLE
Are you working on manifest version 3

### DISCUSSION THREAD

**Discussion by rahulbansal16** [alyssaxuu/screenity#195] [Q&A] [answered]
Hey,
Are you working on manifest version 3?

**Comment by rahulbansal16** [comment]
@alyssaxuu would be really helpful if you provide details about it. I saw your newsletter about you started working on it. 
It will be really helpful if you are porting it to manifest v3, I am thinking to port it.

**Comment by alyssaxuu** [comment, marked as answer]
Yes, I've rebuilt Screenity from zero for MV3. I will be sharing it soon (maybe in a week or two), mostly just testing it at this point so there's no issues.

**Comment by rahulbansal16** [comment]
Will it be available under the same licencse?

**Reply by alyssaxuu** [reply]
I'm still deciding on the license, but it won't remain MIT (obviously the MV2 version of it will always be MIT licensed though). The whole extension code will likely be a GPLv3 license or similar. My main intention for this repo is to allow Screenity to be self-hosted without needing to go through the Chrome Store and give users more control. I've seen a lot of Screenity copycats and rip offs, many with the same branding and features, which is not my desired goal.

I am also exploring creating a separate repo for more of a developer SDK to integrate within different websites, in which case it might have a different license, but I'm not sure yet.

**Comment by guidedtirth** [comment]
please update MV3.

**Comment by slns** [comment]
Hello, I would like to be able to contribute to the project, but I am unable to build the project.
It gives an error after doing:
npm start
ERROR in permissions
Module not found: Error: Can't resolve 'screenity/src/pages/permissions/index.jsx' in 'screenity'

**Reply by alyssaxuu** [reply]
Looks like it was a case sensitivity issue in Webpack. Can you try again to see if it works?

**Reply by slns** [reply]
Thanks

**Comment by rahulbansal16** [comment]
@alyssaxuu 
How are you making sure that the extension has the mic permission on all the websites?

I was checking that you have enabled iframe, allow microphone, camera everywhere, but certain websites like WhatsApp Web block the access. 

How are you overriding that part?
